helpers/joiSchema.js

```diff
@@ -0,0 +1,9 @@
+const Joi = require("joi");
+
+const schema = Joi.object({
+  name: Joi.string().min(3).max(30),
```

Comment:
Taki tip. Jak w środku podasz np email() podasz wiadomość to pojawi się ona zamiast domyślego tekstu :D

Reply:
Próbowałem dodać default-ową wartość maila ale w dokumentacji wyczytałem, że Note that quoted email addresses (e.g. "test"@example.com) are not supported and will fail validation. Zamiast tego dodałem metodę default